client/core/src/com/cyberbot/bomberman/screens/AbstractScreen.java

```diff
@@ -4,13 +4,16 @@
 import com.badlogic.gdx.Screen;
 import com.badlogic.gdx.graphics.GL20;
 import com.cyberbot.bomberman.Client;
+import com.cyberbot.bomberman.controllers.GameScreenController;
 
 public abstract class AbstractScreen implements Screen {
 
     protected final Client app;
+    protected final GameScreenController gameScreenController;
 
-    public AbstractScreen(final Client app) {
+    public AbstractScreen(final Client app, GameScreenController gameScreenController) {
         this.app = app;
```

Comment:
`app` was deemed useless, any reasons to keep it?

Reply:
fixed